src/runtime/runtime.js

```diff
@@ -276,9 +276,9 @@ export class Runtime {
   }
 
   /** @override */
-  getEntitlements(encryptedDocumentKey) {
+  getEntitlements(params) {
```

Comment:
ditto

Reply:
Same as above. I think for backwards compatibility we need to have two arguments here, where the publisher can pass in the encryptedDocumentKey directly or as a part of the params.

Reply:
Disregard! You addressed the concern already.